contentcuration/contentcuration/frontend/channelEdit/components/edit/DetailsTabView.vue

```diff
@@ -117,17 +117,20 @@
               />
             </p>
 
+            <!-- Need to break up v-model to properly show placeholder -->
+
             <!-- Author -->
             <VCombobox
               ref="author"
-              v-model="author"
               :items="authors"
               :label="$tr('authorLabel')"
               :readonly="disableAuthEdits"
               maxlength="200"
-              box
               autoSelectFirst
+              box
               :placeholder="getPlaceholder('author')"
+              :value="author && author.toString()"
```

Comment:
I am not sure if I follow - could you please elaborate?

Reply:
We use an object `{toString: () => ''}` to communicate when multiple items are selected in the list, but they have different values for that field. VTextField interprets this as having some value, so the placeholder doesn't show up. This works around that by just going for the toString method value

Reply:
Thanks for the clarification